python_modules/dagster/dagster/_core/definitions/multi_asset_sensor_definition.py

```diff
@@ -1204,13 +1164,3 @@ def __call__(self, *args, **kwargs):
 
         context.update_cursor_after_evaluation()
         return result
-
-    @public  # type: ignore
```

Comment:
I was thinking that it might make sense to coerce it to a single type, which is what we do in most other places where we accept multiple types. E.g. maybe if someone passes in a collection of asset keys, we should coerce it to `AssetSelection.keys(the_keys)`. But I was going back and forth on this a little, so decided to punt.

Have you observed that these attributes are used by users?

Reply:
I haven't seen it used anywhere. I'm fine with removing! just was curious what the reasoning is